docs_src/microservices/support/scheduler/Ch-Scheduler.md

```diff
@@ -0,0 +1,111 @@
+# Scheduler
+
+![image](EdgeX_SupportingServicesScheduling.png)
+
+## Introduction
+
+The support scheduler micro service provide an internal EdgeX “clock” that can kick off operations in any EdgeX service.  At a configuration specified time (called an **interval**), the service calls on any EdgeX service API URL via REST to trigger an operation (called an **interval action**).  For example, the scheduler service periodically calls on core data APIs to clean up old sensed events that have been successfully exported out of EdgeX.
+
+### Default Interval Actions
+
+Scheduled interval actions configured by default with the reference implementation of the service include:
+
+- Clean up Core-data events/readings that have been moved ("pushed") to external applications/systems like a cloud provider. This is the "Scrubbed Pushed" operation. Scheduler parameters around this operation determine how often and where to call into core data to invoke this clean up of unneeded data operation.
```

Comment:
Remove this as mark as pushed was removed.

Reply:
Updated.